frontend/src/views/DatabaseDetail.vue

```diff
@@ -72,29 +72,6 @@
                 >{{ projectName(database.project) }}</router-link
               >
             </dd>
-            <template v-if="database.sourceBackup">
```

Comment:
We don't need to show this anymore?

Reply:
Yeah, we didn't show it for PITR restored database. We just keep it as a record without showing this information for now.